### DISCUSSION TITLE
Can't install on Ubuntu 22.04 because 'libmpv2'

### DISCUSSION THREAD

**Discussion by vgbach** [KRTirtho/spotube#805] [Q&A]
Hi I'm trying to install Spotube on Ubuntu 22.04 and after running `sudo dpkg -i Spotube-linux-x86_64.deb` the installation won't proceed because a dependency (in this case it was 'libmpv2') is missing and I have no clue where to get it from. Sorry if the answer seems obvious to you I'm new to Linux in general and installing things have been quite a confusing experience.

**Comment by KRTirtho** [comment]
Please use the nightly version of the app. This has been fixed
Or, you can use the Flatpak variant for getting the latest stable version
